todo-frontend-angular/src/app/todolist-selector/todo/todo.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { Todo } from 'src/app/models/todo.model';
+
+@Component({
+  selector: 'app-todo',
+  templateUrl: './todo.component.html',
+  styleUrls: ['./todo.component.scss']
+})
+export class TodoComponent implements OnInit {
+  deadline: string = '';
+  constructor() { }
+
+  @Input() todo?: Todo;
+
+  @Output() onCheckboxClick = new EventEmitter<boolean>();
```

Comment:
This chould be named: checkboxClick then the eventhandler should be named onCheckboxClick :)

Reply:
updated